docs/testing/README.md

```diff
@@ -0,0 +1,26 @@
+# Testing in LocalStack
+
+- [Integration Tests](integration-tests/README.md)
+- [Parity Testing](parity-testing/README.md)
+- [Multi-account and Multi-region Testing](multi-account-region-testing/README.md)
+- [Terraform Tests](terraform-tests/README.md)
+
+## Rules for stable tests
+
+Through experience, we encountered some guiding principles and rules when it comes to testing LocalStack.
+These aim to ensure a stable pipeline and keep flakes minimal. Any newly added test and feature should keep these in mind!
+
+| **ID** | **Rule** |
+|-------|-----------|
+| R01   | Inform code owners and/or test authors when a test is marked as flaky so that it can be fixed ASAP. |
```

Comment:
Maybe we should even directly file a PR marking it as skip? This would automatically inform code owners that a test is flaky 😛

Reply:
That is what I was thinking about concerning the "Inform" part. I will make this more explicit!